src/main/java/cn/nukkit/level/Level.java

```diff
@@ -1754,7 +1753,7 @@ public Item useBreakOn(Vector3i pos, BlockFace face, Item item, Player player, b
             this.updateComparatorOutputLevel(target);
         }
 
-        target.onBreak(item);
+        target.onBreak(item, player);
```

Comment:
This is where the new method is called, methods which doesn't overrides the new method will do the same as before.